Docs/source/usage/parameters.rst

```diff
@@ -1115,8 +1115,16 @@ Laser initialization
     using ``my_constants``. For a two-dimensional simulation, similar to the B-field,
     it is assumed that the first and second dimensions are `x` and `z`, respectively,
     and the value of the `Ey` component is set to zero.
-    The current implementation of the parser for B-field on particles
+    The current implementation of the parser for E-field on particles
     is applied in cartesian co-ordinates as a function of (x,y,z) even for RZ.
+    To apply a series of plasma lenses, use the option ``repeated_plasma_lens``. This
+    option requires the following parameters,
+    ``repeated_plasma_lens_period``, the period length of the repeat, a single float number,
+    ``repeated_plasma_lens_starts``, the start of each lens relative to the period, an array of floats,
```

Comment:
Just curious: is start preferred over mid point of the lens for descriptions?

Reply:
I don't think it matters. Using starts is simplest to code so I chose that.

Reply:
Ok, thanks for the information! :)
Yes, I was wondering from a user's point-of-view what is more common for placing the lenses.